test/hotspot/jtreg/applications/jcstress/README

```diff
@@ -24,10 +24,15 @@ The tests located under this directory run tests from the Java Concurrency
 Stress test suite[1] (a.k.a. jcstress). This suite aims to verify the
 correctness of concurrency support in the JDK.
 
-All the tests are run through the test driver class -- JcstressRunner, which
-downloads the specified build of org.openjdk.jcstress:jcstress-tests-all,
+All the tests are run through the test driver class -- JcstressRunner, 
 spawns a new JVM to run one jcstress test and checks that it finishes
-successfully.
+successfully. Before you start these tests, you should get the specified
+build of org.openjdk.jcstress:jcstress-tests-all, the quickest way of
+get specified build is download the prebuilt JAR from
+[here](https://builds.shipilev.net/jcstress/).
+You should specific the JAR location with jtreg option such as
+-Djdk.test.lib.artifacts.jcstress-tests-all=jcstress-tests-all-20241217-2035.jar
+when start jcstress tests in jtreg.
```

Comment:
```suggestion
You should specify the JAR location by passing the following property when invoking jtreg. For example:
-Djdk.test.lib.artifacts.jcstress-tests-all=jcstress-tests-all-20241217-2035.jar
```
Though I'm unclear if this is a setting passed directly to jtreg, or whether it should be set via e.g. `-javaoption:-Dxxx` ?